Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs

```diff
@@ -142,15 +142,18 @@ private async Task Login(CancellationToken cancellationToken)
                 return;
             }
 
-            var loginResponse = await OpenSubtitlesHandler.OpenSubtitles.LogInAsync(options.Username, options.Password, "en", cancellationToken).ConfigureAwait(false);
+            var loginResponse = await OpenSubtitlesHandler.OpenSubtitles.LogInAsync(
+                options.Username,
+                options.Password,
+                "en",
+                cancellationToken).ConfigureAwait(false);
 
-            if (loginResponse.Item2 != null)
+
+            if (loginResponse.Item2 == 1)
```

Comment:
You added an extra newline here. Also, we dont need to check for a null object anymore?

Reply:
Can’t be null if it’s equal to 1 can it? :p